parlai/core/metrics.py

```diff
@@ -34,6 +34,100 @@
+    "f1": MetricDisplayData(
+        "F1", "Unigram F1 overlap, under a standardized (model-independent) tokenizer"
+    ),
+    "gnorm": MetricDisplayData("Gradient Norm", "Gradient norm"),
+    "gpu_mem": MetricDisplayData(
+        "GPU Memory",
+        "Fraction of GPU memory used. May slightly underestimate true value.",
+    ),
+    "hits@1": MetricDisplayData(
+        "Hits@1", "Fraction of correct choices in 1 guess. (Similar to recall@K)"
+    ),
+    "hits@5": MetricDisplayData(
+        "Hits@5", "Fraction of correct choices in 5 guesses. (Similar to recall@K)"
+    ),
+    "interdistinct-1": MetricDisplayData(
+        "Interdistinct-1", "Fraction of n-grams unique across _all_ generations"
+    ),
+    "interdistinct-2": MetricDisplayData(
+        "Interdistinct-1", "Fraction of n-grams unique across _all_ generations"
+    ),
+    "intradistinct-1": MetricDisplayData(
+        "Intradictinct-1", "Fraction of n-grams unique _within_ each utterance"
+    ),
+    "intradictinct-2": MetricDisplayData(
+        "Intradictinct-2", "Fraction of n-grams unique _within_ each utterance"
+    ),
+    "jga": MetricDisplayData("Joint Goal Accuracy", "Joint Goal Accuracy"),
+    "llen": MetricDisplayData(
+        "Label Length", "Average length of label in number of tokens"
+    ),
+    "loss": MetricDisplayData("Loss", "Loss"),
+    "lr": MetricDisplayData("Learning Rate", "The most recent learning rate applied"),
+    "ltpb": MetricDisplayData("Label Tokens Per Batch", "Label tokens per batch"),
+    "ltps": MetricDisplayData("Label Tokens Per Second", "Label tokens per second"),
+    "ltrunc": MetricDisplayData(
+        "Label Truncation", "Fraction of samples with some label truncation"
+    ),
+    "ltrunclen": MetricDisplayData(
+        "Label Truncation Length", "Average length of label tokens truncated"
+    ),
+    "rouge-1": MetricDisplayData("ROUGE-1", "ROUGE metrics"),
+    "rouge-2": MetricDisplayData("ROUGE-2", "ROUGE metrics"),
+    "rouge-L": MetricDisplayData("ROUGE-L", "ROUGE metrics"),
+    "token_acc": MetricDisplayData(
+        "Token Accuracy", "Token-wise accuracy (generative only)"
+    ),
+    "token_em": MetricDisplayData(
+        "Token Exact Match",
+        "Utterance-level token accuracy. Roughly corresponds to perfection under greedy search (generative only)",
+    ),
+    "total_train_updates": MetricDisplayData(
+        "Total Train Updates", "Number of SGD steps taken across all batches"
+    ),
+    "tpb": MetricDisplayData(
+        "Tokens Per Batch", "Total tokens (context + label) per batch"
+    ),
+    "tps": MetricDisplayData(
+        "Tokens Per Second", "Total tokens (context + label) per second"
+    ),
+    "ups": MetricDisplayData("Updates Per Second", "Updates per second (approximate)"),
+}
+
+
+def get_metric_display_data(metric: str) -> MetricDisplayData:
+    return METRICS_DISPLAY_DATA.get(
+        metric, MetricDisplayData(title=metric, description="No description provided.")
```

Comment:
How about adding  `No description provided. Please add it to metrics.py if this is an official metric in ParlAI.`